pkg/workload/cli/run.go

```diff
@@ -86,10 +87,17 @@ var individualOperationReceiverAddr = sharedFlags.String(
 var histograms = runFlags.String(
 	"histograms", "",
 	"File to write per-op incremental and cumulative histogram data.")
+
+var histogramExportFormat = runFlags.String(
+	"histogram-export-format", "hdr-json",
```

Comment:
I'm ok with json also, I made it hdr-json because people are aware that the exported format is a hdr-json whereas there's a histogram type metric in openmetrics which this change is emitting.